playbooks/dbgsym-kernel.yml

```diff
@@ -0,0 +1,27 @@
+---
+- name: add debug symbol archive key
+  apt_key: keyserver=keyserver.ubuntu.com id=2512191FEF8729D6E5AF414DECDCAD72428D7C01
```

Comment:
#1488 should allow using keyserver/id. This PR will need to be rebased if/after that is merged, so that you can use the role.